### ISSUE TITLE
Register Resolvers

### ISSUE THREAD

**Issue by zachferland** [decentralized-identity/did-jwt#9]
Appears that resolvers can only be registered where `did-jwt` is imported. Does not appear possible to register resolver when using dependency which `did-jwt` is a dependency. 

i.e 

```js
//uport-js 
import { verifyJWT } from 'did-jwt'

const funcUsingVerifyJWT = () => {
   // verifyJWT( ... )
}

export {  funcUsingVerifyJWT }
```

```js
//uport-connect
import { funcUsingVerifyJWT } from 'uport-js'
import registerUport from 'uport-did-resolver'

// Now this doesn't work 
registerUport( registryCustomConfig )
```

Saw this in the past, so above is just an outline of the problem, can also create a working code example again to verify. If this is a problem, can also look closer at code here to propose solution. 

Maybe considered a non-issue as well, saw that uport-js did implementation got around this by just consuming the args to the supported resolvers and then registering them there. But still seems like there are use cases where you want them to work more generally.

**Comment by pelle** [comment]
Lets think about solutions for this

**Comment by mirceanis** [comment]
This was fixed in #57 and released in v4.0.0
A `Resolver` needs to be specified when calling `verifyJWT`
